src/main/java/com/epam/training/geomety/logic/SphereCalculator.java

```diff
@@ -0,0 +1,83 @@
+package com.epam.training.geomety.logic;
+
+import com.epam.training.geomety.entities.Plane;
+import com.epam.training.geomety.entities.Point3D;
+import com.epam.training.geomety.entities.Sphere;
+import org.apache.log4j.Logger;
+
+public class SphereCalculator {
+    private static final Logger LOGGER = Logger.getRootLogger();
+    private static final double PI = Math.PI;
```

Comment:
in this case, you may directly use Math.PI without this constant (your variable is also possible, but a bit unccessesary in practice I would say)